lodash.src.js

```diff
@@ -11204,6 +11226,34 @@
     }
 
     /**
+     * This function is like `property` except it takes a `keys` array to get nested property.
+     * @static
+     * @memberOf _
+     * @category Utility
+     * @param {string[]} keys Keys that specify the property to get.
+     * @returns {Function} Returns the new function.
+     * @example
+     *
+     * var users = [
+     *   { 'user': { 'name': 'fred'   } },
+     *   { 'user': { 'name': 'barney', 'age': 36 } }
+     * ];
+     *
+     * var getName = _.propertyDeep(['user', 'name']);
+     * var getAge  = _.propertyDeep(['user', 'age']);
+     *
+     * _.map(users, getName);
+     * // => ['fred', 'barney']
+     *
+     * _.map(users, getAge);
+     * // => [undefined, 36]
+     */
+    function propertyDeep(keys) {
+      keys = baseMap(keys, function(key) { return key + '' });
```

Comment:
`arrayMap` doesn't allow falsey first argument though. Should I make `propertyDeep` rejectFalsey?